App/Sagas/TextileSagas.js

```diff
@@ -209,7 +209,7 @@ export function * photosTask () {
   try {
     yield call(BackgroundTimer.start)
     const photos = yield call(queryPhotos)
-    for (const photo of photos) {
+    for (const photo of photos.reverse()) {
```

Comment:
this just allows your most recently taken photo to load last when bulk loading new images. it also makes the upload order appear a bit clearer

Reply:
Oh, then you switch it here

Reply:
unrelated. this is just the order within the new photos

Reply:
or, this has nothing to do with display order, right?

Reply:
so they one at a top pop into the displayed array. if you have your _newest_ pop in first, then others have to pop in behind it. that's pretty odd. here the oldest are going to pop in first. let me record you a gif

Reply:
I get that, it's fine. This is a good change. This makes the photos pop in from oldest to newest. I like it.

Reply:
Think we were miscommunicating  about what I meant by 'display order'. I meant the order of photos in the grid as a whole, while you were talking about the order in which they appear when added.